index.bs

```diff
@@ -179,13 +179,25 @@ a number of properties for itself:
 *   <dfn for="user agent" export>full version</dfn> (for example: "72.0.3245.12", "3.14159", or "297.70E04154A")
 *   <dfn for="user agent" export>platform brand</dfn> (for example: "Windows", "iOS", or "AmazingOS")
 *   <dfn for="user agent" export>platform version</dfn> (for example: "NT 6.0", "15", or "17G")
-*   <dfn for="user agent" export>platform architecture</dfn> (for example: "ARM64", or "ia32")
+*   <dfn for="user agent" export>platform architecture</dfn> (for example: "arm", or "x86")
 *   <dfn for="user agent" export>model</dfn> (for example: "", or "Pixel 2 XL")
 *   <dfn for="user agent" export>mobileness</dfn> (for example: ?0 or ?1)
 
 [=User agents=] SHOULD keep these strings short and to the point, but servers MUST accept arbitrary
 values for each, as they are all values constructed at the [=user agent=]'s whim.
 
+[=User agents=] MUST map higher-entropy [=platform architecture=] values to the following buckets:
+
+* x86 CPU architectures => "x86"
+* ARM CPU architectures => "arm"
```

Comment:
I did ask for less of this info, but I think this is so much less that it's not useful any more. In practice, x86_64 and arm64 are different enough architectures that in practice they cannot be considered the same as their 32-bit cousins.

What value does this info provide that the empty string does not?

Not only does this not directly provide value, but I also I think there's a hazard here that x86 vs arm could start to be wrongly used as a desktop vs mobile distinction. Yes, there is a token for this purpose, but I'm hard pressed to think of any other more valid use of a CPU architecture field with only these two values.

Reply:
The general thinking when talking to @bslassey about this was that this would enable 32 bit installers that then can then download the appropriate variant, as a bare minimum, and we could later maybe enable a more accurate CPU reporting if needed. (which would include both the 32/64 bit distinction, as well as potentially other characteristics, like ARM version)

I agree that the 32/64 bit distinction feels more useful than the latter part, so I'm happy to modify the PR to include it. In browsers that would take the Privacy Budget route, it would increase the "price" of this API, but that's probably fine.

Reply:
>  I also I think there's a hazard here that x86 vs arm could start to be wrongly used as a desktop vs mobile distinction.

I'm not sure who would make this assumption.  The Surface Pro X ships with a native ARM version of Edge.  Rumors say other software giants (😉) may soon offer the same.  Chromebooks can run Linux binaries, but they're architecture dependant.  ISVs may want metrics as well.

I'm drafting this into a project for my own analytics (future proofing, anticipating the analytics platforms to slowly deprecate the old in favor of the new) and find the directives hard to follow.  For example, current I convert `amd64` to `x86_64` for the User-Agent string to mimic other systems, but this seems like I should simply use `x86`, which seems like a loss of information.

Reply:
Apologies but I missed your comment. x86 is indeed a loss of information, but that loss is intentional, to enable folks to access only the partial information when it is enough. The hint discussed in https://github.com/WICG/ua-client-hints/issues/105 will enable accessing bitness as well.